docs/development/bug-process/README.md

```diff
@@ -0,0 +1,183 @@
+
+Out of the list of confirmed bugs, the most important / urgent bugs are prioritized to be addressed in the next version. For this, there is a status “prioritized” which means that these are the bugs that the development team looks at to fix in the upcoming versions.
+
+<u>Steps:</u>
+
+- Out of the list of confirmed bugs, the most important bugs are moved from status “confirmed” to “prioritized”.
+  - Ideally, not every bug that could be worked on is moved to the “prioritized” status as it will complicate the actual prioritization, i.e. weighing the importance of fixing one bug against all of the others in the next step.
+- The list of prioritized bugs is sorted based on importance: The most important bug is at the top of the list, the least important bug at the bottom of the list.
+- The prioritized list contains the bugs that the development team can look at next and take bugs from to fix them (see step 4.Assigning bugs to version & fixing bugs).
+  - The bugs at the top of the list are being looked at first (note: due to technical affinity, complexity, dependencies, etc. this does not mean that these bugs are necessarily fixed in that order)
+
+
+There is a work package view that reflects those prioritized bugs: https://community.openproject.org/projects/openproject/work_packages?query_id=2865
+
+![bug-prioritization-list](bug-prioritization-list.png)
+
+
+
+The bugs are assessed and prioritized mostly according to their severity. The effort required to fix them may also be a factor but as no thorough technical analysis will take place up until after the prioritization has already taken place, there will not be a dependable estimate on the effort.  
+
+Bugs can only be set to “prioritized” by OpenProject team members. When assigning bugs to “prioritized”, they should be prioritized against all other bugs. Therefore, team members who assign bugs to “prioritized” should have an overview of the other bugs to evaluate how critical the issue really is. Ideally, the assignment to “prioritized'' should be done by team members who have a good overview of existing bugs. Most of the time, this should be carried out by QA/Community Management.
+
+QA, Community Management scans the “prioritized” list in regular intervals to adjust the order and add additional bugs. 
+
+Other team members can add bugs as well. (Example: There may be bugs which are more technical and may hinder development while not directly affecting users. This may not be clear to QA but only to Software Development.) Ideally, the necessity to prioritize a bug fix is talked through with QA/Community Management, to ensure that the “prioritized” list includes the most important bugs.
+
+The work package view with the prioritized bugs does not include bugs that are outside the scope of an OpenProject version (e.g. changes to the website, changes to SaaS, etc.)
+
+
+
+### 4. Assigning bugs to version & fixing bugs
+
+<u>Involved teams:</u>
+
+- Software Development
+
+<u>Summary</u>:
```

Comment:
That whole block has changed in the mean time. With our continuous bug fixing process they way we work on the prioritised bugs is different.